### PR TITLE
Transfer: cleanup payment id code

### PR DESCRIPTION
**Pull request by tobtoht**
Test with #2626

### CODE REVIEW

**Comment by selsta** [comment]
The long PID code is still used for when pasting e.g.

```
monero:46BeWrHpwXmHDpDEUmZBWZfoQpdc6HaERCNmx1pEYL2rAcuwufPN9rXHHtyUA4QVy66qeFQkn6sfK8aHYjA3jk3o1Bv16em?tx_amount=239.39014&tx_description=donation&tx_payment_id=0f33ad024c7788e623662a4b1461699225848ea90948a783a37e261f2d55c087
```

**Comment by tobtoht** [comment]
Thanks, will fix.

**Comment by tobtoht** [comment]
Hmm, what should the expected behavior be here, keep as is?

In that case the "remove after pid removal hardfork" comments should be removed.

**Comment by selsta** [comment]
> Hmm, what should the expected behavior be here, keep as is?

yes

> In that case the "remove after pid removal hardfork" comments should be removed.

yea they probably should get removed

File: pages/Transfer.qml [hunk 1]
```diff
@@ -403,17 +387,17 @@ Rectangle {
...
+                  error: paymentIdCheckbox.checked
               }
           }
       }
 
       MoneroComponents.WarningBox {
-          // @TODO: remove after pid removal hardfork
           id: paymentIdWarningBox
-          text: qsTr("Long payment IDs are obsolete. \
+          text: qsTr("Long payment IDs are no longer supported. \
```
Comment (selsta) [inline pages/Transfer.qml]: Please change it back to obsolete, we lose the translation otherwise.

**Pull request merged**